src/utils.ts

```diff
@@ -37,3 +37,14 @@ export function urlGenerator(nextCode?: string): string {
 export function unfollowUserUrlGenerator(idToUnfollow: string): string {
     return `https://www.instagram.com/web/friendships/${idToUnfollow}/unfollow/`;
 }
+
+/**
+ * @returns Array of unique members from two given arrays.
+ *
+ * https://en.wikipedia.org/wiki/Set_(abstract_data_type)
+ */
+export function uniqueMerge(arr1: readonly any[], arr2: readonly any[]): readonly any[] {
+    const mergedArray = [...arr1, ...arr2];
+    const set = new Set(mergedArray);
+    return [...set];
+}
```

Comment:
Ended up not needing this, but left it in there anyways, might prove useful at some point.